### PR TITLE
adding svg's for n-ring and benzene ring

### PR DESCRIPTION
**Pull request by KranthiMedikonda**

### CODE REVIEW

File: public/svg/benzene-ring.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (AaronLlanos) [inline public/svg/benzene-ring.svg]: please get rid of all comments, tittle, desc, and ids in this file

File: public/svg/n-ring.svg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <title>Rings/n-ring</title>
```
Comment (AaronLlanos) [inline public/svg/n-ring.svg]: same here

File: public/svg/benzene-ring.svg [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 52.6 (67491) - http://www.bohemiancoding.com/sketch -->
+    <title>Rings/benzene-ring</title>
+    <desc>Created with Sketch.</desc>
+    <g id="Rings/benzene-ring" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```
Comment (AaronLlanos) [inline public/svg/benzene-ring.svg]: we are going to need to make sure the fill and stroke don't need to be removed. Usually, these items will affect highlighting on select

File: src/localization/en/kekule.localize.widget.en.js [hunk 4]
```diff
@@ -20,6 +20,8 @@ const reversibleArrowLine = require("../../../public/svg/reversible-arrow-line.s
 const openArrowDoubleLine = require("../../../public/svg/open-arrow-double-line.svg");
 const heatSymbol = require("../../../public/svg/triangle.svg");
 const addSymbol = require("../../../public/svg/plus.svg");
+const Benzene = require("../../../public/svg/benzene-ring.svg");
```
Comment (AaronLlanos) [inline src/localization/en/kekule.localize.widget.en.js]: please follow variable naming convention camelCase

Comment (KranthiMedikonda) [reply src/localization/en/kekule.localize.widget.en.js]: @AaronLlanos   will address all of them in the next commit :)

**Pull request closed without merging**